app/styles/scenariobuilder.scss

```diff
@@ -33,6 +33,35 @@
     padding: 8px 16px; // Extra pixel top and bottom for border on buttons
    }
 
+  .toggle {
+    margin-top: $padding;
+  }
+
+  .multiplytable {
```

Comment:
If the goal is to make something full-width, maybe rename this to something like "full-width"?